web/modules/custom/rdf_entity/src/Entity/RdfEntityType.php

```diff
@@ -0,0 +1,63 @@
+<?php
+/**
+ * @file
+ * Defines the Rdf entity type: e.g. Asset, Interoperability Solution,...
+ */
+
+namespace Drupal\rdf_entity\Entity;
+
+use Drupal\Core\Config\Entity\ConfigEntityBundleBase;
+use Drupal\rdf_entity\RdfEntityTypeInterface;
+
+
+/**
+ * Defines the Rdf models.
+ *
+ * @ConfigEntityType(
+ *   id = "rdf_type",
+ *   label = @Translation("Rdf entity type"),
+ *   handlers = {
+ *     "list_builder" = "\Drupal\rdf_entity\Entity\Controller\RdfTypeListBuilder",
+ *     "form" = {
+ *       "default" = "\Drupal\rdf_entity\Form\RdfTypeForm",
+ *       "reset" = "Drupal\taxonomy\Form\VocabularyResetForm",
```

Comment:
Heh. `Drupal\taxonomy\Form\VocabularyResetForm` :)

There is also a route defined that shows the form to reset the vocabulary to alphabetical ordering: `entity.rdf_type.reset_form`. This should also be removed from the routing definitions I guess.